.github/workflows/release-drafter.yml

```
@@ -1,4 +1,30 @@
+      - uses: actions/checkout@v2
+      - name: Update Release Draft
+        uses: release-drafter/release-drafter@v5
+        with:
+          config-name: release-drafter.yml
+
+  publish_release:
+    needs: update_release_draft
+    if: startsWith(github.ref, 'refs/tags/')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)